lms/static/js/learner_dashboard/views/certificate_view.js

```diff
@@ -0,0 +1,30 @@
+;(function (define) {
+    'use strict';
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/certificate.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             certificateTpl
+         ) {
+            return Backbone.View.extend({
+                el: '.certificates-list',
+                tpl: _.template(certificateTpl),
+                initialize: function(certificatesData) {
+                    this.certificatesData = certificatesData;
```

Comment:
I think it makes sense to call ```this.render()``` in the initialize function so when instantiating the view you do not need to append .render(). The view will always be rendered after initialization so I would prefer to keep instantiation a little cleaner.